### DISCUSSION TITLE
Prohibited from use in China

### DISCUSSION THREAD

**Discussion by JameChen** [rustdesk/rustdesk#7952] [General]
Prohibited from use in China. I just need to set up a local area network for learning. What should I do

**Comment by JameChen** [comment]
How to do it in detail, do you have a tutorial? It is currently disabled. I am deploying Docker as a continuation server on my own computer, which is not an external network and cannot be used. It is very unfriendly to Chinese users

**Reply by rustdesk** [reply]
China is one of regions with the worst scamming, many seniors lost their money of whole family because of this, do you know how desperate they are? They may kill themself because money lost.

I have no detail, since you are learning, hope you can figure it out, it is also a part of learning.

**Reply by wyyw5701207** [reply]
Okay, I haven't seen the scenery outside the wall before. Is it really that bad here

**Reply by nadirvishun** [reply]
@rustdesk  You can take a look at how many fraud cases there are in the United States in 2023: [As Nationwide Fraud Losses Top $10 Billion in 2023, FTC Steps Up Efforts to Protect the Public](https://www.ftc.gov/news-events/news/press-releases/2024/02/nationwide-fraud-losses-top-10-billion-2023-ftc-steps-efforts-protect-public). Shouldn't you ban the entire American region? Or even further, shut down the entire internet. Contrary to what you said, a quick search will show that China is currently the country with the strictest anti-telecom fraud laws. Not long ago, they even took down an entire fraud organization from Vietnam. Isn't it better to do some research before speaking or acting?

**Reply by Ekcum** [reply]
@rustdesk it sounds a little racist ._.
scamming in one region doesnt allow to disable the service there, dont u agree?
the scammers will just use some other software.
ig there is another reason

**Reply by Gallium70** [reply]
> @rustdesk You can take a look at how many fraud cases there are in the United States in 2023: [As Nationwide Fraud Losses Top $10 Billion in 2023, FTC Steps Up Efforts to Protect the Public](https://www.ftc.gov/news-events/news/press-releases/2024/02/nationwide-fraud-losses-top-10-billion-2023-ftc-steps-efforts-protect-public). Shouldn't you ban the entire American region? Or even further, shut down the entire internet. Contrary to what you said, a quick search will show that China is currently the country with the strictest anti-telecom fraud laws. Not long ago, they even took down an entire fraud organization from Vietnam. Isn't it better to do some research before speaking or acting?

They have an announcement written in Chinese [here](https://www.oschina.net/news/291123), and I assume you can read it. I believe they did more research than you had expected.

**Comment by rustdesk** [comment]
Now, we only restrict Android devices being controlled.

**Reply by wyyw5701207** [reply]
3Q，But being able to control Android is really convenient. Will this be considered open
